tests/Browser/Pages/Backend/Borrows/SearchBorrowsTest.php

```diff
@@ -0,0 +1,160 @@
+
+    }
+
+     /**
+     * A Dusk test looking input search null with select name
+     *
+     * @return void
+     */
+    public function testSeeInputNullSelectName()
+    {
+        $this->makeData(6);
+        $this->browse(function (Browser $browser) {
+           $browser->loginAs($this->user)
+                    ->visit('/admin/borrows')
+                    ->resize(1200,1600)
+                    ->assertSee('List Borrower')
+                    ->assertInputValue('search', '')
+                    ->select('filter', 'name')
+                    ->click('#btn-search')
+                    ->visit('/admin/borrows?search=&filter=name')
+                    ->assertQueryStringHas('search', '')
+                    ->assertQueryStringHas('filter', 'name')->screenshot(1);
+            $elements = $browser->elements('#list-borrows tbody tr');
+            $this->assertCount(7, $elements);
+        });
+    }
+
+    /**
+     * A Dusk test looking input search null with select title
+     *
+     * @return void
+     */
+    public function testSeeInputNullSelectBook(){
+        $this->makeData(6);
+        $this->browse(function (Browser $browser){
+            $browser->loginAs($this->user)
+                    ->visit('/admin/borrows')
+                    ->resize(1200,1600)
+                    ->assertSee('List Borrower')
+                    ->assertInputValue('search', '')
+                    ->select('filter', 'title')
+                    ->click('#btn-search')
+                    ->visit('/admin/borrows?search=&filter=title')
+                    ->assertQueryStringHas('search', '')
+                    ->assertQueryStringHas('filter', 'title')->screenshot(2);
+            $elements = $browser->elements('#list-borrows tbody tr');
+            $this->assertCount(7, $elements);
+        });
+    }
+
+    /**
+     * A Dusk test when input search username incorrect with select name
+     *
+     * @return void
+     */
+    public function testSearchSelectName(){
+        $this->makeData(6);
+        $this->browse(function (Browser $browser){
+           $browser->loginAs($this->user)
+                    ->visit('/admin/borrows')
+                    ->resize(1200,1600)
+                    ->assertSee('List Borrower')
+                    ->type('search', 'Minh Dao T.')
+                    ->select('filter', 'name')
+                    ->click('#btn-search')
+                    ->visit('/admin/borrows?search=Minh Dao T.&filter=name')
+                    ->assertQueryStringHas('search', 'Minh Dao T.')
+                    ->assertQueryStringHas('filter', 'name')->screenshot(1);
+            $elements = $browser->elements('#list-borrows tbody tr');
+            $this->assertCount(1, $elements);
+        });
+    }
+
+    /**
+     * A Dusk test when input search titlebook incorrect with select title
+     *
+     * @return void
+     */
+    public function testSearchSelectBook(){
+        $this->makeData(6);
```

Comment:
function nào cũng dùng `$this->makeData(6)` thì anh nên đặt lên Setup() luôn,không phải gọi lại